File: development/OnBoarding/OnBoarding/OnBoarding.Web/src/app/model/associateProjectGroup.ts
export interface IAssociateProjectGroup {
    Id: string,
    AssociateGroupId: string,
    Code: string,
    Name: string,
    RoleId: string,
    IsGroup: boolean
}

export class AssociateProjectGroup implements IAssociateProjectGroup {

    constructor(
        public Id: string,
        public AssociateGroupId: string,
        public Code: string,
        public Name: string,
        public RoleId: string,
        public IsGroup: boolean ) {
    }
}

File: development/OnBoarding/OnBoarding/OnBoarding.Contract/Associate/AssociateProjectGroup.cs
namespace OnBoarding.Contract
{
    using System;
    public class AssociateProjectGroup
    {
        public Guid Id { get; set; }

        public Guid? AssociateGroupId { get; set; }

        public string Name { get; set; }

        public string Code { get; set; }

        public Guid RoleId { get; set; }    
          
        public bool IsGroup { get; set; }
    }
}
